node.gyp

```diff
@@ -178,7 +178,6 @@
 
       'dependencies': [
         'node_js2c#host',
-        'deps/nghttp2/nghttp2.gyp:nghttp2'
```

Comment:
@mmarchini I don't quite understand why this was added.  It doesn't seem to be used.

Reply:
This shouldn't be here, it was accidentally added when the branch was rebased